ingestion/src/metadata/cmd.py

```diff
@@ -130,88 +131,93 @@ def report(config: str) -> None:
         ) from exc
 
 
+# metadata docker --start
+# metadata docker --start -f <path_to_docker_compose_file>
+# metadata docker --stop
+# metadata docker --pause
+# metadata docker --resume
+
+
 @metadata.command()
 @click.option("--start", help="Start release Docker containers", is_flag=True)
-@click.option("--stop", help="Stop Docker containers (local and release)", is_flag=True)
 @click.option(
-    "--clean",
-    help="Prune unused containers, images, volumes and networks",
+    "--stop",
+    help="Stop and remove openmetadata Docker containers (local and release) and images",
     is_flag=True,
 )
+@click.option("--pause", help="Pause openmetadata Docker containers", is_flag=True)
 @click.option(
-    "-t",
-    "--type",
-    help="'local' - local type will start local build of OpenMetadata docker",
-    default="release",
-    required=False,
+    "--resume", help="Resume/Unpause openmetadata Docker containers", is_flag=True
 )
 @click.option(
-    "-p",
-    "--path",
+    "-f",
+    "--file-path",
     help="Path to Local docker-compose.yml",
     type=click.Path(exists=True, dir_okay=False),
     required=False,
 )
-def docker(start, stop, clean, type, path) -> None:
+def docker(start, stop, pause, resume, file_path) -> None:
     """
     Checks Docker Memory Allocation
     Run Latest Release Docker - metadata docker --start
-    Run Local Docker - metadata docker --start -t local -p path/to/docker-compose.yml
+    Run Local Docker - metadata docker --start -f path/to/docker-compose.yml
     """
 
     try:
-        import docker as sys_docker
+        # import python_on_whales as sys_docker
+        from python_on_whales import DockerClient
 
         from metadata.ingestion.ometa.ometa_api import OpenMetadata
         from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
 
-        client = sys_docker.from_env()
```

Comment:
lets move this code into its own file docker.py and make it a method that can be imported here

Reply:
Done. Thanks Harsha